views/js/previewer/plugins/content/scoringHighlighter.js

```diff
@@ -89,6 +70,58 @@ define([
                 })
             );
 
+            /**
+             * Gets the renges of the selection
+             * 
+             * @param {*} selection 
+             */
+            function getAllRanges(selection) {
+                const allRanges = [];
+
+                for (let i = 0; i < selection.rangeCount; i++) {
+                    allRanges.push(selection.getRangeAt(i));
+                }
+                return allRanges;
+            }
+
+            /**
+             * Erases highlights and updates highlightIndex
```

Comment:
`* Erases highlights and notifies the parent iframe`?

Reply:
better :)